src/GeositeFramework/js/SubRegion.js

```diff
@@ -58,17 +75,44 @@
     }
 
     function addSubRegionsToMap(subregions, layer) {
-
         _.each(subregions.areas, function(subregion) {
             var geom = new esri.geometry.Polygon(subregion.shape);
             var symbol = new esri.symbol.SimpleFillSymbol();
             symbol.setColor(new dojo.Color(subregion.color || subregions.color));
 
-            var graphic = new esri.Graphic(geom, symbol);
-            graphic.attributes = subregion;
+            var infoTemplate = new esri.InfoTemplate();
+            infoTemplate.setContent("${display}");
```

Comment:
`display` is the name of the field that contains the subregion name. I'll add a comment.